### ISSUE TITLE
Frames must have an accessible name

### ISSUE THREAD

**Issue by susiestrath** [PlaneteStream/moodle-filter_planetestream#3]
Hi team,

I hope I am raising the issue in the right repository, apologies if this isn't the case.

We are getting this accessibility failure when running the Axe dev tools against pages that have an eStream embeds: "Frames must have an accessible name".

This is a violation of [WGAC 2.1 SC 4.1.2: Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).

To fix the issue, the iframe element should be given a "title" attribute with the title of the video as a value. Please note that only the title OR the aria-label attribute needs to be added, not both.

![CleanShot 2024-04-15 at 11 18 30](https://github.com/PlaneteStream/moodle-filter_planetestream/assets/79458670/173ee9a1-4492-45bd-bc2c-667b314edcf9)

Is there any chance this could be fixed please?
